example/lib/src/screen/home_page.dart

```diff
@@ -18,6 +18,8 @@ class _HomePageState extends State<HomePage>
 
   final Set<String> _selectedScopes = Set.from(["profile"]);
 
+  final _codeEditingController = TextEditingController();
```

Comment:
no need to add this as an option in sample app, since it's only for Android platform, and is not a must configuration.